src/CakeManager.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Migrations;
+
+use Phinx\Migration\Manager;
+
+/**
+ * Overrides Phinx Manager class in order to provide an interface
+ * for running migrations within an app
+ */
+class CakeManager extends Manager {
```

Comment:
Any reason you are using CakeManager and not Manager with aliasing of the original class?